### PR TITLE
Fix a11y on generate new OTP link in contacts

### PR DESCRIPTION
**Pull request by NicolaGiornetta**
## Short description
Fix a11y on generate new OTP link in contacts.

## List of changes proposed in this pull request
- Made accessible navigation on link "generate new OTP" in contacts OTP modal

## How to test
Now you should be able to navigate via TAB key to "generate new OTP" link.

### CODE REVIEW

File: packages/pn-personafisica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx [hunk 1]
```diff
@@ -285,15 +286,14 @@ const DigitalContactsCodeVerificationProvider: FC<ReactNode> = ({ children }) =>
...
                 &nbsp;
               </Typography>
-              <Typography
-                variant="body2"
-                display="inline"
-                color="primary"
+              <ButtonNaked
                 onClick={() => handleCodeVerification(undefined, true)}
-                sx={{ cursor: 'pointer' }}
+                sx={buttonNakedInheritStyle}
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx:291]: Maybe this style is useless. Add only verticalAlign: unset and check that mobile and desktop views are ok

File: packages/pn-personagiuridica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx [hunk 2]
```diff
@@ -285,15 +286,14 @@ const DigitalContactsCodeVerificationProvider: FC<ReactNode> = ({ children }) =>
...
                 &nbsp;
               </Typography>
-              <Typography
-                variant="body2"
-                display="inline"
-                color="primary"
+              <ButtonNaked
                 onClick={() => handleCodeVerification(undefined, true)}
-                sx={{ cursor: 'pointer' }}
+                sx={buttonNakedInheritStyle}
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx:291]: The same as before

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-personafisica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx (hunk 1 above)
Comment (NicolaGiornetta) [reply packages/pn-personafisica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx]: Done

File: packages/pn-personagiuridica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx (hunk 2 above)
Comment (NicolaGiornetta) [reply packages/pn-personagiuridica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx]: Done

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**